### DISCUSSION TITLE
ARM64 iso download

### DISCUSSION THREAD

**Discussion by murech** [flatcar/Flatcar#800] [Q&A]
The iso image is on the [release page](https://stable.release.flatcar-linux.net/) available for amd64 but not for arm64. Where can I find an iso image?

**Comment by jepio** [comment]
hi @murech, the amd64 iso currently only works on non-uefi systems and relies on syslinux (amd64 only). We do not have an iso for arm64 systems.

You should be able to take the https://stable.release.flatcar-linux.net/arm64-usr/current/flatcar_production_qemu_uefi_image.img.bz2 file, uncompress it and write it to a CD or USB stick and boot that on any supported arm64 system. Let me know how that works.
